src/app/app.module.ts

```diff
@@ -3,11 +3,15 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { ProductsComponent } from './modules/products/products.component';
+import { ProductsModule } from './modules/products/products.module';
+import { ProductsService } from './modules/products/products.service';
+import { ProductComponent } from './modules/product/product.component';
 
 @NgModule({
-  declarations: [AppComponent],
-  imports: [BrowserModule, AppRoutingModule],
-  providers: [],
+  declarations: [AppComponent, ProductsComponent, ProductComponent],
```

Comment:
You don't need to import it into app.module, because it is a separate module with routing and it will be loaded separately in the routing, and there is no need to put it in import app.module.ts.